FreshRSS/FreshRSS
# Add error handling for tag creation failure in addAction method

Closes #6887 

Changes proposed in this pull request:

- Add error handling in `addAction()` to verify if `addTag()` succeeded.
- Show a failure message when `addTag()` fails instead of always showing a success message.

How to test the feature manually:

1. Attempt to create a new tag with a unique name and confirm it shows a success message.
2. Attempt to create a tag with a duplicate name and confirm it shows the "Label cannot have the same name as Category" message.
3. Induce a failure in `addTag()` (e.g., by temporarily causing a database error) to see if the failure message appears as expected.

Pull request checklist:
